index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet"/>
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="hamburge-menu">
+        <a href="#">John Doe</a>
+        <img src="./img/Menu.png" alt="Hamburger-Menu" />
+      </nav>
+    </header>
+    <main>
+      <div class="message">
+        <h1>Hey There. <br />I’m Alex</h1>
```

Comment:
- [ ] Please make sure that your `<h1>` tag has the same font-weight and font-size and also line spacing as defined on Figma template that will make your website wonderful :smile: